app/helpers/plantings_helper.rb

```diff
@@ -0,0 +1,15 @@
+module PlantingsHelper
+
+  def display_planting(planting)
+    if planting.quantity.to_i > 0 && planting.planted_from.present?
```

Comment:
I think `quantity` is already an integer so we shouldn't need `to_i` here or below.